src/app/core/services/refiner.service.ts

```
@@ -248,7 +250,9 @@ export class RefinerService {
       eou: this.authService.getEou(),
       homeCurrency: this.currencyService.getHomeCurrency(),
       deviceInfo: Device.getInfo(),
-    }).subscribe(({ isConnected, eou, homeCurrency, deviceInfo }) => {
+      clusterDomain: this.tokenService.getClusterDomain(),
+    }).subscribe(({ isConnected, eou, homeCurrency, deviceInfo, clusterDomain }) => {
+      console.log(clusterDomain);
```

ESLint

[error] Unexpected console statement.

(no-console)